evap/staff/templates/staff_course_operation.html

```diff
@@ -20,11 +20,17 @@
             </tbody>
         </table>
         {% if show_email_checkbox %}
+        <div class="form-group">
             <div class="checkbox">
                 <label>
                     <input type="checkbox" name="send_email" checked> {% trans "Send email notifications" %}
                 </label>
             </div>
+            <label>Email subject:</label> <br>
+            <input type="text" name="email_subject" value="{{ email_template.subject}}" style="width: 100%;"> <br>
+            <label>Email body:</label>
+            <textarea class="form-control" name="email_body" style="resize:vertical;min-height:300px;"> {{ email_template.body }} </textarea>
+        </div>
         {% endif %}
         <button type="submit" name="operation" value="{{ operation }}" class="btn btn-primary">{% trans "Confirm" %}</button>
```

Comment:
- please put the buttons into a `<div class="well submit-area text-center">` (see email pages of courses)
- and please use the form design we have on other pages (e.g. the email page of courses), where form labels are besides the input fields and have no colon.